modules/passkey/hardhat.config.ts

```diff
@@ -74,6 +74,9 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: ETHERSCAN_API_KEY,
   },
+  sourcify: {
+    enabled: false,
```

Comment:
Why false? i thought sourcify is superior to etherscan

Reply:
Brainfart. Should be fixed now.